cinnamon-remix/modules/finished.conf

```diff
@@ -0,0 +1,3 @@
+---
+restartNowMode: user-checked
+restartNowCommand: "systemctl -i reboot"
```

Comment:
@fossfreedom - we had a short conversation about this; was this the solution we came up with?

Reply:
remix will need to look at unmounting the cdrom before rebooting or at least something like that to ensure a clean reboot.